dev-tools/osx-deploy

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import os
+import argparse
+from subprocess import check_call
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Builds OSX packages for Beats")
+    args = parser.parse_args()
```

Comment:
Probably this should have the same option as `dev-tools/deploy`?

https://github.com/elastic/beats/blob/96b5aeeaa776dcfe2252209c3633d6339d94592e/dev-tools/deploy#L9-L10